api-scanner/src/main/java/net/corda/plugins/ApiPrintWriter.java

```diff
@@ -0,0 +1,128 @@
+package net.corda.plugins;
+
+import io.github.lukehutch.fastclasspathscanner.scanner.ClassInfo;
+import io.github.lukehutch.fastclasspathscanner.scanner.FieldInfo;
+import io.github.lukehutch.fastclasspathscanner.scanner.MethodInfo;
+import io.github.lukehutch.fastclasspathscanner.typesignature.TypeSignature;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.lang.reflect.Modifier;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+import static java.util.stream.Collectors.joining;
+import static java.util.stream.Collectors.toCollection;
+
+public class ApiPrintWriter extends PrintWriter {
+    ApiPrintWriter(File file, String encoding) throws FileNotFoundException, UnsupportedEncodingException {
+        super(file, encoding);
+    }
+
+    public void println(ClassInfo classInfo, int modifierMask, List<String> filteredAnnotations) {
+        super.append(asAnnotations(filteredAnnotations, ""));
```

Comment:
General comment: Why `super.append()` instead of just `append()`? You haven't overridden the `append` function in this class, but you'd presumably want to invoke your own own rather than the parent one, if you had.